.travis.yml

```diff
@@ -6,5 +6,6 @@ node_js:
   - "0.10"
 after_script:
   - npm run coveralls
+  - npm run lint
```

Comment:
Travis will now run the `lint` task and fail the build if any errors were found.

Unrelated, but not sure if you want to drop "iojs" builds and/or add "4.0" (aka "stable") builds to Travis. Let me know and I can append my PR or create a new one.